internal/otelcollector/config/metric/gateway/processors.go

```diff
@@ -92,6 +92,16 @@ func makeDropIfInputSourceIstioConfig() *FilterProcessor {
 	}
 }
 
+func makeDropIfEnvoyMetricsDisabledConfig() *FilterProcessor {
+	return &FilterProcessor{
+		Metrics: FilterProcessorMetrics{
+			Metric: []string{
+				ottlexpr.IsMatch("name", "^envoy_.*"),
```

Comment:
we should assure the metric is emmitted by us by checking the instrumentation scope

Reply:
the condition extended with istio scope check